src/frontend/admin/src/services/api/models/CertificateDefinition.ts

```diff
@@ -13,3 +13,7 @@ export type DTOCertificateDefinition = Optional<CertificateDefinition, "id">;
 export type CertificateDefinitionFormValues = ToFormValues<
   Omit<CertificateDefinition, "id">
 >;
+
+export enum CertificationDefinitionTemplate {
+  Certificate = "certificate",
```

Comment:
By convention, as enum are constant we use key in uppercased.
```suggestion
  CERTIFICATE = "certificate",
  DEGREE = "degree",
```